test/integration.js

```diff
@@ -365,6 +365,14 @@ test('ChatList methods', t => {
   t.end()
 })
 
+test('key transfer methods', t => {
+  dc.initiateKeyTransfer((err, setupCode) => {
+    t.same(err, null, 'err is null')
```

Comment:
Actually, we should use the same thing for errors, e.g. `t.error(err, 'no error')`, see https://github.com/substack/tape#terrorerr-msg

Reply:
I should have said this from the start, sorry about that.

Reply:
> I should have said this from the start, sorry about that.

no problem ^^